homework/hw01/task3.py

```diff
@@ -0,0 +1,24 @@
+# 3. Напишите код, который запрашивает число и сообщает является ли оно простым
+# или составным. Используйте правило для проверки:
+# “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+MAX_NUMB = 100000
+def is_prime(n):
+    if n <= 1:
+        return False
+    for i in range(2, int(n ** 0.5) + 1):
+        if n % i == 0:
+            return False
+    return True
+
+while True:
+    number = int(input('Введите число: '))
+    if number < 0 or number > MAX_NUMB:
+        print('Число должно быть положительным и не больше 100 тысяч.')
+    else:
+        if is_prime(number):
+            print(f'{number} - простое число')
+        else:
+            print(f'{number} - составное число')
+        break
```

Comment:
Можно цикл оставить только для получения переменной `number`
```suggestion
while True:
    number = int(input('Введите число: '))
    if 0 <= number <= MAX_NUMB:
        break
        
    print('Число должно быть положительным и не больше 100 тысяч.')

if is_prime(number):
    print(f'{number} - простое число')
else:
    print(f'{number} - составное число')
```